lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionHandler.java

```diff
@@ -222,32 +221,24 @@ private void saveQuestionStateToKbvItem(
 
     private QuestionsResponse getQuestionAnswerResponse(
             KBVItem kbvItem, SessionItem sessionItem, Map<String, String> requestHeaders)
-            throws JsonProcessingException, SqsException {
+            throws SqsException {
         Objects.requireNonNull(kbvItem, "kbvItem cannot be null");
 
-        if (kbvItem.getExpiryDate() == 0L) { // first request for questions for a given session
```

Comment:
Can you confirm that this condition never evaluated to true? While in production if this was ever triggered then we'll need to be capturing this case too?